airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -21,6 +21,10 @@
 from sqlalchemy import or_
 
 from airflow._vendor.connexion import NoContent
+from airflow.api.common.experimental.mark_tasks import (
+    set_dag_run_state_to_failed,
+    set_dag_run_state_to_success,
+)
```

Comment:
Can we do without using experimental API ?

Reply:
Hi Kaxil, I think the utility functions inside` common/experimental` package are not yet deprecated but the experimental REST API is? The experimental API lives in `api/client` package.  I tried not using these functions but the change was big. Now I'm suggesting we use it but move the useful util functions out of the `/experimental` or rename the `/experimental` to api_utils or something appropriate?

Reply:
There are actually quite a few places depending on `airflow.api.common.experimental` in the web UI and the new API. We should do a more comprehensive refactoring to move those out of `experimental` instead, so I feel it should be OK-ish to just referring them right here.